sql.html.markdown

```diff
@@ -5,7 +5,7 @@ contributors:
   - ["Bob DuCharme", "http://bobdc.com/"]
 ---
 
-Structured Query Language (SQL) is an ISO standard language for creating and working with databases stored in a set of tables. Implementations usually add their own extensions to the language; [Comparison of different SQL implementations](http://troels.arvin.dk/db/rdbms/) is a good reference on product differences.
+Structured Query Language (SQL) is an [ISO/IEC 9075](https://www.iso.org/standard/53686.html) standard language for creating and working with databases stored in a set of tables. Implementations usually add their own extensions to the language; [Comparison of different SQL implementations](http://troels.arvin.dk/db/rdbms/) is a good reference on product differences.
```

Comment:
this seems to be a more fitting link to me, what do you think? https://www.iso.org/standard/63555.html

Reply:
Agreed!